### ISSUE TITLE
Support .net8

### ISSUE THREAD

**Issue by zepa19** [thecloudnativewebapp/OidcProxy.Net#108]
When version that supports .net8 will be published?

**Comment by appie2go** [comment]
I can publish it any time. Shall I publish it next monday? I wanted to add some more features, because this is going to be a breaking change. But i assume you need it?

**Comment by zepa19** [comment]
It's not urgent for me. I was only checking dependencies of my project and saw, that this library is not yet published for .net8. I  will do a migration of my project when all dependencies supports .net8

**Comment by appie2go** [comment]
Hi there, i'm about to implement a solution for the following two issues, and then i intend to release an RC for v1.0.0. Keep u posted:

* https://github.com/thecloudnativewebapp/GoCloudNative.Bff/issues/80
* https://github.com/thecloudnativewebapp/GoCloudNative.Bff/issues/84

**Comment by appie2go** [comment]
Hi there,

I just published v1.0.0-rc.1. It contains quite some changes, not all of them have been documented. I would appreciate it if you would have a look at it to see if it fits your needs.

https://www.nuget.org/packages/GoCloudNative.Bff.Authentication.OpenIdConnect/1.0.0-rc.1

Cheers,

**Comment by appie2go** [comment]
Hi all,

Thanks for contributing to this project, by providing your feedback. As you may have noticed, we've decided to rebrand the product to OidcProxy.Net. There is now a v1.0.0 available on nuget.

The docs have mostly been updated, and i'll keep updating some more. Most important changes are the bootstrap the app, which now works like this:

```csharp
using OidcProxy.Net.ModuleInitializers;
using OidcProxy.Net.OpenIdConnect;

var builder = WebApplication.CreateBuilder(args);

var config = builder.Configuration
    .GetSection("OidcProxy")
    .Get<OidcProxyConfig>();

builder.Services.AddOidcProxy(config);

var app = builder.Build();

app.UseOidcProxy();

app.Run();
```

Any feedback or pull request because of incomplete docs are appreciated!

Cheers!
